api/messages/index.js

```diff
@@ -0,0 +1,79 @@
+import connectToMongodb from '../_database/connect-to-mongodb';
+import {Message} from '../_database/models';
+
+const Handler = async (request, response) => {
+  switch (request.method) {
+    case 'GET':
+      try {
+        await connectToMongodb();
+        const messages = await Message.find({});
+        if (messages) {
+          return response.json(
+            messages.map(message => {
+              return {_id: message._id, msg: message.msg};
+            })
+          );
+        } else {
+          const error = new Error({message: 'Message not known', status: 500});
+          throw error;
+        }
+      } catch (err) {
+        console.log(err);
+        return response.status(500).json({message: 'Server error'});
```

Comment:
This is more or less the same, isn't it? Is there a "better" error you can throw in line 17-18?
I think you might rather want to say something like "sorry, there are no messages yet" or similar if `messages` is not truthy.

Reply:
good idea